packages/backend/src/graphql/resolvers/outgoing_payment.ts

```diff
@@ -107,9 +121,21 @@ export const createOutgoingPayment: MutationResolvers<ApolloContext>['createOutg
     const outgoingPaymentService = await ctx.container.use(
       'outgoingPaymentService'
     )
-    const outgoingPaymentOrError = await outgoingPaymentService.create(
-      args.input
-    )
+
+    const tenantId = ctx.forTenantId
+    if (!tenantId)
+      throw new GraphQLError(
+        `Assignment to the specified tenant is not permitted`,
+        {
+          extensions: {
+            code: GraphQLErrorCode.BadUserInput
+          }
+        }
+      )
+    const outgoingPaymentOrError = await outgoingPaymentService.create({
+      tenantId,
+      ...args.input
```

Comment:
```suggestion
      ...args.input,
      tenantId
```
just in case - so we don't override the calculated tenantId